rpcs3/Crypto/unself.cpp

```diff
@@ -890,8 +899,9 @@ bool SELFDecrypter::DecryptNPDRM(u8 *metadata, u32 metadata_size)
 	memcpy(klicensee_key, key_v.GetKlicenseeKey(), 0x10);
 
 	// Use klicensee if available.
-	if (klicensee_key != NULL)
-		memcpy(npdrm_key, klicensee_key, 0x10);
+	// FIXME: Check is always false.
+	/*if (klicensee_key != NULL)
+		memcpy(npdrm_key, klicensee_key, 0x10);*/
```

Comment:
@Nekotekina What do you mean? This check is **always** false, thus I commented it and added a "FIXME" note. If you want me to uncomment me it, I can do that.